traffic_ops/app/lib/API/ApiCapability.pm

```diff
@@ -0,0 +1,258 @@
+package API::ApiCapability;
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+#
+
+use UI::Utils;
+
+use Mojo::Base 'Mojolicious::Controller';
+use Data::Dumper;
+
+
+
+my $finfo = __FILE__ . ":";
+
+my %valid_http_methods = map { $_ => 1 } ('GET', 'POST', 'PUT', 'PATCH', 'DELETE');
+
+sub all {
+	my $self = shift;
+	my @data;
+	my $orderby = "capability";
+	$orderby = $self->param('orderby') if ( defined $self->param('orderby') );
+
+	my $rs_data = $self->db->resultset("ApiCapability")->search( undef, { order_by => $orderby } );
+	while ( my $row = $rs_data->next ) {
+		push(
+			@data, {
+				"id"          	=> $row->id,
+				"httpMethod"	=> $row->http_method,
+				"route" 		=> $row->route,
+				"capName"   	=> $row->capability->name,
+				"lastUpdated" 	=> $row->last_updated
+			}
+		);
+	}
+	$self->success( \@data );
+}
+
+
+sub renderResults {
+	my $self = shift;
+	my $rs_data = shift;
+
+	my @data = ();
+	while ( my $row = $rs_data->next ) {
+		push(
+			@data, {
+				"id"          	=> $row->id,
+				"httpMethod"	=> $row->http_method,
+				"route" 		=> $row->route,
+				"capName"   	=> $row->capability->name,
+				"lastUpdated" 	=> $row->last_updated
+			}
+		);
+	}
+	$self->success( \@data );
+}
+
+sub capName {
```

Comment:
see my comment in trafficopsroutes.pm. This action can go away as it can be rolled into the index action and coupled with a query param to get the same results.